topostats/grains.py

```diff
@@ -126,7 +126,7 @@ def calc_minimum_grain_size(self, image: np.array) -> float:
         """
         self.get_region_properties()
         grain_areas = np.array([grain.area for grain in self.region_properties])
-        grain_areas = grain_areas[grain_areas > threshold(grain_areas, method=self.threshold_method)]
+        grain_areas = grain_areas[grain_areas >= threshold(grain_areas, method=self.threshold_method)]
```

Comment:
Nice when it's this simple!

Reply:
If only everything in life were that simple!

> One thing I would recommend, either as part of this PR, or a later one, would be to include as test to prove that the exception is raised when appropriate data is thrown at it. `with pytest.raises` etc.

:+1: Definitely, I'm awaiting some images with zero grains so I can write tests to identify what exceptions are raised and check them in tests.